From 2140fcff04b004fa1a6ef54feb35e12655659869 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 5 Feb 2015 17:54:30 +0100
Subject: [PATCH 017/310] lirc: update to 0.9.2

- no kernel.mk needed (since 0.9.1+)
- remove obsolete configure switches
- add PKG_INSTALL
- add PKG_FIXUP since autotools are used

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/lirc/Makefile | 20 +++++---------------
 1 file changed, 5 insertions(+), 15 deletions(-)

diff --git a/utils/lirc/Makefile b/utils/lirc/Makefile
index 541f56c..cdcc5b9 100644
--- a/utils/lirc/Makefile
+++ b/utils/lirc/Makefile
@@ -1,22 +1,22 @@
 #
-# Copyright (C) 2006-2013 OpenWrt.org
+# Copyright (C) 2006-2015 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
 #
 
 include $(TOPDIR)/rules.mk
-include $(INCLUDE_DIR)/kernel.mk
 
 PKG_NAME:=lirc
-PKG_VERSION:=0.8.7
+PKG_VERSION:=0.9.2
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=@SF/${PKG_NAME}
-PKG_MD5SUM:=fb1dcddbc8ca3d03c7f3a2d40cf28b71
+PKG_MD5SUM:=3afc84e79c0839823cc20e7a710dd06d
 
-PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
+PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -58,10 +58,7 @@ define Package/lirctools/description
 endef
 
 CONFIGURE_ARGS += \
-	--enable-sandboxed \
-	--with-driver="mceusb" \
 	--with-gnu-ld \
-	--with-kerneldir="$(LINUX_DIR)" \
 	--with-pic \
 	--without-x \
 	--disable-nls \
@@ -71,9 +68,6 @@ define Package/lirc/conffiles
 /etc/lircd.conf
 endef
 
-MAKE_FLAGS += \
-	DESTDIR="$(PKG_INSTALL_DIR)" \
-	SUBDIRS="daemons tools"
 
 define Package/lirc/install
 	$(INSTALL_DIR) $(1)/usr/sbin/
@@ -81,10 +75,6 @@ define Package/lirc/install
 	$(INSTALL_DIR) $(1)/etc
 	$(INSTALL_CONF) $(PKG_BUILD_DIR)/contrib/lircd.conf $(1)/etc/
 endef
-#todo:
-#	$(CP) $(PKG_BUILD_DIR)/drivers/lirc_dev/lirc_dev.ko
-#	$(CP) $(PKG_BUILD_DIR)/drivers/lirc_mceusb2/lirc_mceusb2.ko
-#depends on the selected module
 
 define Package/lircdaemonadd/install
 	$(INSTALL_DIR) $(1)/usr/sbin/
-- 
2.5.0

